app/controllers/schools/partnerships_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Schools::PartnershipsController < Schools::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
+
+  def index
+    @school = current_user.induction_coordinator_profile.schools.first
+    @partnership = @school.partnerships&.find_by(cohort: cohort)
+    @delivery_partner = @partnership&.lead_provider&.delivery_partners&.find_by(provider_relationships: { cohort: cohort })
```

Comment:
That sounds too wide. The lead provider will have many, many delivery partners which have this cohort in their relationships.

Reply:
I'm filtering by school as well. A school can only have 1 partnership with lead provider / delivery partner per cohort.

Reply:
```
@delivery_partner = @partnership&.lead_provider&.delivery_partners&.find_by(provider_relationships: { cohort: cohort })
```
`@partnership` is definitely filtered by school. Its `lead_provider` doesn't remember which school it was though, I think. So its delivery partners won't be filtered.

Am I missing something?

Reply:
@kicferk1 I've added a migration to add delivery partners to a partnership. That seems to be the way forward as in the proposed designs delivery partners get selected as part of the workflow for getting into a partnership. https://miro.com/app/board/o9J_ldVNkCY=/?moveToWidget=3074457355944253374&cot=14

Reply:
Alright, that makes sense. Then this line would change to `@delivery_partner = @partnership.delivery_partner`?